app/org/sagebionetworks/bridge/models/healthdata/HealthDataRecord.java

```diff
@@ -15,17 +15,20 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.joda.time.format.DateTimeFormat;
+import org.joda.time.format.DateTimeFormatter;
 
 /** This class represents health data and associated metadata. */
 @BridgeTypeName("HealthData")
 @JsonDeserialize(as = DynamoHealthDataRecord.class)
 public interface HealthDataRecord extends BridgeEntity {
+    DateTimeFormatter TIME_ZONE_FORMATTER = DateTimeFormat.forPattern("Z");
```

Comment:
Just checking that this will parse all 4 of the acceptable formats:
<time>Z
<time>±hh:mm
<time>±hhmm
<time>±hh